### PR TITLE
[guilib][PVR] Fix some compiler warnings

### PR DESCRIPTION
**Pull request by ksooo**
Some compiler warning I came across recently.

Runtime-tested on macOS and Android, latest Kodi master.

@phunkyfish when you find time for a review.

### CODE REVIEW

File: xbmc/pvr/timers/PVRTimers.cpp [hunk 1]
```diff
@@ -589,15 +589,15 @@ bool CPVRTimers::UpdateEntries(int iMaxNotificationDelay)
             if (nextStart.IsValid())
             {
               bool bCreate = false;
-              const auto it = m_tags.find(nextStart);
-              if (it == m_tags.end())
+              const auto it1 = m_tags.find(nextStart);
```
Comment (howie-f) [inline xbmc/pvr/timers/PVRTimers.cpp:592]: `it1`? why not `tag`

Comment (ksooo) [reply xbmc/pvr/timers/PVRTimers.cpp:592]: it's not a tag, it's an iterator

**Comment by lrusak** [comment]
jenkins build this please

**Review by phunkyfish — APPROVED** [review]
All sane

**Pull request merged**